.github/workflows/publish_charm.yaml

```diff
@@ -161,7 +161,7 @@ jobs:
           rm -rf $TEMP_DIR
       - name: Get charm name
         id: get-charm-name
-        run: echo "CHARM_NAME=$(yq '.name' metadata.yaml)" >> $GITHUB_OUTPUT
+        run: echo "CHARM_NAME=$([ -f metadata.yaml ] && yq '.name' metadata.yaml)" >> $GITHUB_ENV
```

Comment:
edge case: if .name does not exist, CHARM_NAME will be "null". Is it worth using something like "yq -e 1 " so CHARM_NAME will be empty in this scenario too?

Reply:
Won't the current code have the same effect?